src/schema/v2/order/sharedOrderTypes.ts

```diff
@@ -415,6 +428,15 @@ export const OrderType = new GraphQLObjectType<OrderJSON, ResolverContext>({
         }
       },
     },
+    seller: {
+      type: SellerType,
+      description: "The seller of the order",
+      resolve: ({ seller_id }, _args, { partnerLoader }) => {
+        if (!seller_id) return null
+
+        return partnerLoader(seller_id).catch(() => null)
+      },
```

Comment:
non-blocking: For completeness, should we verify seller type and log/return null if it is not a seller?

Reply:
I'd hope the validation can be provided upstream, but yeah I assume eventually we'll have some logic in place for different seller types.